.github/workflows/nodejs.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x]
+        node-version: [13.x]
```

Comment:
I think we should stick with an LTS version of Node.

Reply:
There are some specific features only supported on 13.x, for example async hooks, and they are really important in logging, which in this case i didn't have implemented yet.

Reply:
Async hooks are still in experimental state for both 12.x and 13.x! Don't thin we should rely on them.

Reply:
Well it was closed on https://github.com/nodejs/node/pull/31945 and released upstream in 13.10.0 https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V13.md#13.10.0
Need to wait for documentation to update.